pkg/kubelet/util/sliceutils/sliceutils_test.go

```diff
@@ -19,10 +19,11 @@ package sliceutils
 import (
 	"testing"
 
-	"k8s.io/api/core/v1"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
```

Comment:
is this line really required?

Reply:
yes, to use `v1.Pod`

Reply:
sorry, i meant to ask should it be just `k8s.io/api/core/v1` or `core_v1 k8s.io/api/core/v1`?

Reply:
Yes, please revert this change.